systests/ws-security/src/test/resources/org/apache/cxf/systest/ws/ut/plaintext-pass-policy.xml

```diff
@@ -2,27 +2,6 @@
 <wsp:Policy xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd" xmlns:wsp="http://www.w3.org/ns/ws-policy" wsu:Id="TransportUsernameTokenPolicy">
     <wsp:ExactlyOne>
         <wsp:All>
```

Comment:
Couldn't we add a new policy rather that modifying an existing testcase?

Reply:
Hi @coheigea,

Revised testcase accordingly.

Cheers
Freeman